zivid_camera/src/zivid_camera.cpp

```diff
@@ -518,11 +518,11 @@ sensor_msgs::PointCloud2ConstPtr ZividCamera::makePointCloud2(const std_msgs::He
   msg->is_dense = false;
 
   msg->fields.reserve(5);
-  msg->fields.push_back(createPointField("x", 0, 7, 1));
-  msg->fields.push_back(createPointField("y", 4, 7, 1));
-  msg->fields.push_back(createPointField("z", 8, 7, 1));
-  msg->fields.push_back(createPointField("c", 12, 7, 1));
-  msg->fields.push_back(createPointField("rgb", 16, 7, 1));
+  msg->fields.push_back(createPointField("x", 0, sensor_msgs::PointField::FLOAT32, 1));
+  msg->fields.push_back(createPointField("y", 4, sensor_msgs::PointField::FLOAT32, 1));
+  msg->fields.push_back(createPointField("z", 8, sensor_msgs::PointField::FLOAT32, 1));
+  msg->fields.push_back(createPointField("c", 12, sensor_msgs::PointField::FLOAT32, 1));
+  msg->fields.push_back(createPointField("rgb", 16, sensor_msgs::PointField::FLOAT32, 1));
```

Comment:
Nice!
Why is the "rgb" field not `sensor_msgs::PointField::UINT8, 3`?

Reply:
This is consistent with how it was before, type number 7. I think it is for PCL compatibility reasons. I just changed to using the enum here instead of 7.

Reply:
Ok, I guess we can do something like [pcl::fromROSMsg(const sensor_msgs::PointCloud2 & cloud, pcl::PointCloud< T > &pcl_cloud)](http://docs.ros.org/kinetic/api/pcl_conversions/html/namespacepcl.html#af662c7d46db4cf6f7cfdc2aaf4439760) to confirm. 
Maybe you did already?

Reply:
I did the following to check and PCL also uses datatype 7 for rgb

  pcl::PointCloud<pcl::PointXYZRGB> cloud;
  sensor_msgs::PointCloud2 pc2;
  toROSMsg(cloud, pc2);

  ROS_INFO_STREAM("Stian debugging!");
  for (const auto& f : pc2.fields)
  {
    ROS_INFO_STREAM(f.name << " datatype=" << std::to_string(f.datatype) << " offset=" << std::to_string(f.offset));
  }

[ INFO] [1597404894.444920693]: x datatype=7 offset=0
[ INFO] [1597404894.444942774]: y datatype=7 offset=4
[ INFO] [1597404894.444961469]: z datatype=7 offset=8
[ INFO] [1597404894.444979463]: rgb datatype=7 offset=16

Reply:
Nice :+1: